.github/workflows/test-runner-nightly-integration.yml

```diff
@@ -64,11 +65,11 @@ jobs:
         run: |
           cd workspace-manager-clienttests
           echo "Upload test results to Google Bucket"
-          ./gradlew uploadResults --args="CompressDirectoryToBucket.json /tmp/TestRunnerResults"
+          ./gradlew uploadResults --args="CompressDirectoryToTerraKernelK8S.json /tmp/TestRunnerResults"
 
       - name: Upload test results to BigQuery Test Runner Dataset
         if: ${{ always() }}
         run: |
           cd workspace-manager-clienttests
           echo "Upload test results to BigQuery Test Runner Dataset"
-          ./gradlew uploadResults --args="SummariesToBigQuery.json /tmp/TestRunnerResults"
\ No newline at end of file
+          ./gradlew uploadResults --args="SummariesToTerraKernelK8SBigQuery.json /tmp/TestRunnerResults"
```

Comment:
It looks like the tables in `terra-kernel-k8s:test_runner_results` are still empty. Have you been uploading to somewhere else in testing?

Reply:
Results are in `testRun` table now, just did a test run to confirm.